config/routes.rb

```diff
@@ -22,4 +22,10 @@
   # resources :posts, only: [:destroy]
 
   resources :comments, only: [:destroy]
+
+  namespace :api, defaults: { format: :json } do
+    resources :posts, only: [:index] do
+      resources :comments, only: [:create, :index]
+    end
+  end
```

Comment:
- [ ] Kindly implement authentication to your API endpoints, both for registration and login.
- Login endpoint - **POST** : `/login`
- Registration endpoint - **POST** : `/register` or `/signup`

Reply:
@oloomoses  thank you for the review. In project requirements, we need to implement it on the post and comment controllers. So first we generate jwt token when login then access this page.